### DISCUSSION TITLE
Introducing RavenDB on Udemy could become be a driver for adoption

### DISCUSSION THREAD

**Discussion by famschopman** [ravendb/ravendb#18891] [Ideas]
![image](https://github.com/user-attachments/assets/dae7df1c-9a2f-46e0-a7f8-db38719f579f)

**Comment by ayende** [comment]
I agree that this would be nice to have. We'll look into that.
In the meantime, there are a number of such courses in PluralSight. 

There is also a free 4+ hours workshop here:
https://ravendb.net/learn/webinars/ravendb-no-sql-document-database-tutorial-dec-2017
